### PR TITLE
Remove slider if light doesn't support brightness

### LINKED ISSUE #3542: Lovelace: Light card shows dimming slider on non-dimming lights.
*[move[bot]](https://github.com/apps/move) commented on Mar 12, 2019, 2:41 AM UTC:*

_[rradillen](https://github.com/rradillen) commented on Feb 24, 2019, 8:12 PM UTC:_

**Home Assistant release with the issue:**
0.86.4

**Browser and Operating System:**
Firefox 65.0.1/Chrome 72.0.3626.119

**Description of problem:**
In the lovelace ui a 'light' card configured with a light entity that is not dimmable still has the dimming slider. I'd expect the dimming slider not to be shown. The entity button is not a valid alternative as it cannot be used in a horizontal stack in conjunction with dimmable lights as the UI elements are not the same height.

_This issue was moved by [iantrich](https://github.com/iantrich) from [home-assistant/home-assistant-polymer#2838](https://github.com/home-assistant/home-assistant-polymer/issues/2838)._

*This issue was moved by [iantrich](https://github.com/iantrich) from [home-assistant/ui-schema#252](https://github.com/home-assistant/ui-schema/issues/252).*

### PR DESCRIPTION
**Pull request by iantrich**
image

I've lost what little CSS mojo I had...HELP! Card is not taking up full space and it is not immediately obvious to me as to why, well because CSS is devil's work :)

Fixes https://github.com/home-assistant/home-assistant-polymer/issues/3542

### CODE REVIEW

File: src/panels/lovelace/cards/hui-light-card.ts [hunk 1]
```diff
@@ -127,27 +129,31 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
...
 
   protected async firstUpdated(): Promise<void> {
-    const loaded = await loadRoundslider();
-
-    this._roundSliderStyle = loaded.roundSliderStyle;
-    this._jQuery = loaded.jQuery;
-
     const stateObj = this.hass!.states[this._config!.entity] as LightEntity;
 
+    if (supportsFeature(stateObj, SUPPORT_BRIGHTNESS)) {
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-light-card.ts]: This is not good, as this means that it's not loaded if the user would change the entity ID (for example in the live editor)

**Comment by balloob** [comment]
If the height is not correct, it is probably because the icon is not taking up their height. Can happen if icon container has a fixed height or if the icon is positioned absolute.

File: src/panels/lovelace/cards/hui-light-card.ts (hunk 1 above)
Comment (iantrich) [reply src/panels/lovelace/cards/hui-light-card.ts]: Yeah that makes sense

**Comment by iantrich** [comment]
I'm going to scrap this for now knowing now that this will change drastically since learning that @thomasloven has a LitElement slider now :tada:

**Pull request closed without merging**